### DISCUSSION TITLE
How to play multiple sozi presentations sequentially on mouse click

### DISCUSSION THREAD

**Discussion by diyoyo** [senshu/Sozi#610] [Q&A]
Hi, 
I would like to segment my sozi deck into multiple html files.
But I need a way to redirect to the next file when one has reached the last slide.
So far I included button images with href links to the next item, but I would love to be able to trigger the redirection on the final mouse click.
What should I modify?
Thanks.
Best,
Y.

PS: bonus points if this is possible from the presenter's view too.

**Comment by aumouvantsillage** [comment]
My usual advice is to use hyperlinks: https://sozi.baierouge.fr/pages/tutorial-links.html
If you want to chain two presentations A and B, you can create a link in the last frame of A and set its `href` attribute to the file `B.sozi.html`.
Some users create a big "button" that fills the whole screen to make sure the click has the expected effect.

At the moment, there is no immediate solution to switch to another presentation without a link.
I guess it could be done by adding a custom script to each presentation, and force the browser to load a new presentation when the current one loops back.
